### PR TITLE
docs: step input component into patterns

### PR DESCRIPTION
**Pull request by jbadan**
## Related Issue
Closes SAP/fundamental-styles#

## Description

Fundamental react had a contributor create a "StepInput" component that matches design specs using existing Fundamental Styles styles - https://github.com/SAP/fundamental-react/pull/914

Adding this patterns page to the documentation website so the libraries stay in sync.

### CODE REVIEW

File: docs/_data/sidebars/left-navigation-sidebar.yml [hunk 1]
```diff
@@ -66,6 +66,10 @@ entries:
       url: /patterns/search-input.html
       output: web
 
+    - title: Step Input
+      url: /patterns/step-input.html
+      output: web
+
```
Comment (jbadan) [inline docs/_data/sidebars/left-navigation-sidebar.yml]: How do we make the pictures for the index page?

Comment (droshev) [reply docs/_data/sidebars/left-navigation-sidebar.yml]: @estelaxu3 do you think you can help in this?

Comment (InnaAtanasova) [reply docs/_data/sidebars/left-navigation-sidebar.yml]: I make them in Adobe XD :D :D :D But I am terrible at this

**Comment by bcullman** [comment]
these would be more useable if the buttons were both on the same side. is this possible?

**Comment by jbadan** [comment]
@bcullman that wouldn't match the design spec though

**Comment by bcullman** [comment]
have the designers actually tried using the stuff they design? :)

we should bring it up with them

**Pull request merged**